src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php

```diff
@@ -321,4 +324,15 @@ public function setOrganizationNameEn(string $organizationNameEn): void
     {
         $this->organizationNameEn = $organizationNameEn;
     }
+
+    public function getInviteRoleId(): ?int
+    {
+        return $this->inviteRoleId;
+    }
+
+    public function registerInvite(string $urn, int $roleId): void
+    {
+        $this->teamName = $urn;
+        $this->inviteRoleId = $roleId;
+    }
```

Comment:
Lovely, this is what I should have done more often in the past. Create expressive methods that are implementing the actual business logic. :+1: